recipes/nspyre/meta.yaml

```diff
@@ -0,0 +1,87 @@
+{% set name = "nspyre" %}
+{% set version = "0.3.0rc1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://github.com/nspyre-org/nspyre/archive/v0.3.0.rc1.tar.gz
+  sha256: e09596836871d5ab993e270971c7287dbb128614977bb6caf81fc2e9bf67f68d
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - nspyre = nspyre.gui:main
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: True
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+  run:
+    - python >=3.8
+    - pip
+    - git
+    - hub
+    # SciPy
+    - numpy >=1.19.1
+    - scipy >=1.5.2
+    - pandas >=1.1.1
+    # MongoDB
+    - defaults::mongodb =4.2.*
```

Comment:
please don't rely on defaults directly

Reply:
hi @scopatz, per a discussion on the conda-forge gitter last week about the lack of availability of a Windows distribution of MongoDB on conda-forge (technical limitations to building for windows), I was told it's okay to list dependencies that are on other channels. I've listed the defaults channel explicitly so that mongoDB is installed from the same source regardless of platform. If you still think this is a bad idea, I'm happy to correct, just let me know.

Reply:
Yeah, it is ok to deps on defaults.  However, this should be implicit so that if/when the deps are added to conda-forge, this package will pick up the conda-forge version.  Having an explicit channel name here prevents that

Reply:
Also can this be `>=4.2`

Reply:
thanks for the explanation and for catching the versioning! both are fixed